lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -45,11 +46,13 @@ def ask_question_loop
     end
 
     def activate
-      if remove_key(force)
-        if fetch_key?
-          fetch_key
+      if !key_exist? || force
+        success_get_new = fetch_key? ? fetch_key : create_key
```

Comment:
I think this would read a bit better if we move the `fetch_key? ? fetch_key : create_key` into a private method. Then we won't need the local variable.

Maybe one called `#get_new_key`?